app/src/firebase/java/org/mozilla/focus/utils/AdjustHelper.java

```diff
@@ -0,0 +1,14 @@
+/* -*- Mode: Java; c-basic-offset: 4; tab-width: 20; indent-tabs-mode: nil; -*-
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.utils;
+
+import android.content.Context;
+
+public class AdjustHelper {
```

Comment:
Firebase should not know about Adjust, right?

Reply:
No. But a dummy implementation is still needed for firebase build type.